internal/pkg/cli/svc_package.go

```diff
@@ -99,34 +103,56 @@ func newPackageSvcOpts(vars packageSvcVars) (*packageSvcOpts, error) {
 		addonsWriter:     ioutil.Discard,
 		fs:               &afero.Afero{Fs: afero.NewOsFs()},
 	}
-
+	appVersionGetter, err := describe.NewAppDescriber(vars.appName)
+	if err != nil {
+		return nil, fmt.Errorf("new app describer for application %s: %w", vars.name, err)
+	}
 	opts.stackSerializer = func(mft interface{}, env *config.Environment, app *config.Application, rc stack.RuntimeConfig) (stackSerializer, error) {
 		var serializer stackSerializer
-		switch v := mft.(type) {
+		switch t := mft.(type) {
 		case *manifest.LoadBalancedWebService:
 			if app.RequiresDNSDelegation() {
-				serializer, err = stack.NewHTTPSLoadBalancedWebService(v, env.Name, app.Name, rc)
+				if err := validateAlias(aws.StringValue(t.Alias), app, env.Name, appVersionGetter); err != nil {
```

Comment:
Since this is the same code as that chunk in `svc_deploy.go`, I wonder if we should extract them out as one function so that we don't need to modify two places in the case of code changes in the future  😢

Reply:
good call!